src/index.html

```diff
@@ -5,8 +5,424 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home"
+        class="header__icon">
+        <img
```

Comment:
If there are only 1 or 2 attributes, then there is no need to transfer them to new lines, this should be corrected
```suggestion
      <a href="#home" class="header__icon">
        <img
```